### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by robertbreker**
- Drop deprecated pip --use-mirrors parameter
- Drop flake8 --show-pep8 as there is no such option anymore
- Fix pep8 complains

### CODE REVIEW

**Comment by lindig** [comment]
I'd like a Python expert to review this but I take it as a good sign that the Travis build works.

File: volume/org.xen.xapi.storage.ffs/tests/test_plugin.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import unittest
 
 sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
-import plugin
+import plugin  # noqa
```
Comment (mseri) [inline volume/org.xen.xapi.storage.ffs/tests/test_plugin.py]: Can you use `# noqa: E402` instead?

File: volume/org.xen.xapi.storage.ffs/tests/test_sr.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from . import TestSrnotattachedexception
 
 sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
-import sr
+import sr  # noqa
```
Comment (mseri) [inline volume/org.xen.xapi.storage.ffs/tests/test_sr.py]: As above

File: volume/org.xen.xapi.storage.ffs/tests/test_volume.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 import util
 
 sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
-import volume
+import volume  # noqa
```
Comment (mseri) [inline volume/org.xen.xapi.storage.ffs/tests/test_volume.py]: As above

File: volume/org.xen.xapi.storage.ffs/tests/test_plugin.py (hunk 1 above)
Comment (robertbreker) [inline volume/org.xen.xapi.storage.ffs/tests/test_plugin.py]: Good idea!

**Pull request merged**